test/e2e/usersignup_test.go

```diff
@@ -596,7 +597,6 @@ func (s *userSignupIntegrationTest) newUserSignup(name string) *v1alpha1.UserSig
 	require.True(s.awaitility.T, ok, "KubeFedCluster should exist")
 
 	spec := v1alpha1.UserSignupSpec{
-		UserID:        uuid.NewV4().String(),
```

Comment:
if we removed the UUID generation of UserID, wouldn't be good to do it for the `ObjectMeta.Name` https://github.com/codeready-toolchain/toolchain-e2e/pull/24/files#diff-cc443128665ec194918207e25d44b3eeR607 instead of using the same given name?